UnitTest1.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MICP_Assignments;
+
+namespace Unit_Test_for_Assignment_6
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        [TestMethod]
+        public void TestMethod1()
+        {
+            BinaryTree tree = new BinaryTree('p', new BinaryTree('a'), new BinaryTree('c'));
+            BinaryTree tree_s = new BinaryTree('p', new BinaryTree('a'), new BinaryTree('c'));
+            Assert.AreEqual(true, Assignment_Week_6.subtreeComparison(tree, tree_s));
+        }
+    }
+}

```

Comment:
It's great to have UTs. I recommend you write one UT per example.

Reply:
even if you don't have time to write all the tests, I think it would be a good idea to just put down some of the equivalence class cases you would test in the comments. :)